services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -145,160 +154,238 @@ public void setUp() {
      */
     @Test
     public void processRequestToSubmitAllSuccess() {
-        final String sdtRequestRef = "TEST_1";
+        final String sdtRequestRef = TEST_1;
```

Comment:
Remove the sdtRequestRef variable - the TEST_1 constant can be used in place of it.

Reply:
done

Reply:
Done